backend/src/tasks/LookingGlass.ts

```diff
@@ -0,0 +1,119 @@
+import {
+  Organization,
+  Domain,
+  Service,
+  Vulnerability,
+  connectToDatabase
+} from '../models';
+import getIps from './helpers/getIps';
+import { CommandOptions } from './ecs-client';
+import got from 'got';
+import { plainToClass } from 'class-transformer';
+import saveVulnerabilitiesToDb from './helpers/saveVulnerabilitiesToDb';
+import { Object } from 'aws-sdk/clients/appflow';
+
+const username: string = "username";
+const passcode: string = "password";
+// let url:string = '/auth/login'
+
+async function getAuth() {
+  const results: any[] = await got
+    .post('https://delta.lookingglasscyber.com' + '/auth/login', {
+      json: { params: { username: username, password: passcode } },
+      headers: {
+        'Content-Type': 'application/json, application/transit+json',
+        Accept: 'application/json, application/transit+json',
+        'User-Agent': 'CISA-VulnerabilityManagement'
```

Comment:
Let's actually remove this User-Agent header. The worker's proxy will automatically add in the Crossfeed User-Agent / signature to the request.